src/components/ControlBar.tsx

```diff
@@ -33,12 +33,12 @@ const ControlBar = () => {
   };
 
   return (
-    <div id="control-bar" className="control-bar">
-      <button className="btn-control" onClick={toggleAudio}>
+    <div className="jlab-gather-control-bar">
```

Comment:
We use the ID to select the control bar [here](https://github.com/QuantStack/jupyterlab-gather/pull/14/files#diff-6f8cf1ecedd4b91fe1c67a66f68b6cf7f587c1bf9edc9935cde80085546d7302R15)